caikit/interfaces/runtime/data_model/info.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file contains interfaces to handle information requests
+"""
+
+# Standard
+from typing import Dict, Optional
+
+# First Party
+import alog
+
+# Local
+from caikit.core.data_model import PACKAGE_COMMON, DataObjectBase, dataobject
+
+log = alog.use_channel("RUNTIMEOPS")
+
+RUNTIME_PACKAGE = f"{PACKAGE_COMMON}.runtime"
+
+
+@dataobject(RUNTIME_PACKAGE)
+class RuntimeInfoRequest(DataObjectBase):
+    pass
```

Comment:
If you put a docstring here, you don't need `pass`

```suggestion
   """Empty request for runtime server information"""
```